src/common/resources/exportConfigs.js

```diff
@@ -0,0 +1,25 @@
+import { get } from 'lodash';
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import {
+  EXPORT_CONFIGURATIONS_API,
+} from '../constants';
+
+export const exportConfigsResource = {
+  ...baseManifest,
+  path: EXPORT_CONFIGURATIONS_API,
+  records: 'exportConfigs',
+  accumulate: true,
+  fetch: false,
+};
+
+export const credentialsResource = {
+  ...baseManifest,
+  accumulate: true,
+  fetch: false,
+  path: (queryParams, pathComponents, resourceData) => {
+    const exportConfigId = get(resourceData, 'exportConfigId.id');
```

Comment:
I see it's coupled with specific resource names, so it doesn't look common to me, is it?

Reply:
Right, ‘exportConfigId’ is created as local resource for storing Id and using it for PUT/POST credentials

Reply:
My concern is that common code shouldn't rely on _magic_ resource names. Usually we do this directly in the component's manifest, around with related resources, so it's obvious that resources a coupled and easier to make changes.